src/command_modules/azure-cli-interactive/azclishell/telemetry.py

```diff
@@ -33,13 +33,19 @@ class Telemetry(TelemetryClient):
     start_time = None
     end_time = None
 
+    @_user_agrees_to_telemetry
     def track_ssg(self, gesture, cmd):
         """ track shell specific gestures """
-        self.track_event('Shell Specific Gesture', {gesture: scrub(cmd)})
+        self.track_event('az/interactive/gesture', {gesture: scrub(cmd)})
+        telthread = TelThread(self.flush)
+        telthread.start()
```

Comment:
We need to rethink about this. We can't start a new thread for telemetry whenever a gesture happens.